dev-packages/cli/README.md

```diff
@@ -56,11 +56,15 @@ If you set `next` in your theia config, then Theia will prefer `next` over `late
 
 To build once:
 
-    theia build
+    theia build --mode development
```

Comment:
The main case: the person creates package.json and use `theia-cli` to build and start it, not to develop something.

Reply:
What are you trying to say? How should I modify this?

Reply:
It should be just `theia build`.

Reply:
Then what's the point of having watch documented just below? I have clearly indicated what to use for production, there's no need for additional changes.

Reply:
ok, i will update docs as the follow up to this PR